content/getting-started/one-off-tasks.md

```diff
@@ -0,0 +1,79 @@
+---
+menu:
+  main:
+    parent: advanced
+title: Running One Off Tasks
+aliases:
+- /getting-started/cf-ssh/
+---
+
+There are a couple of ways to run one off tasks in Cloud Foundry. The most reliable way to do one-off tasks is by deploying a short-lived app.
+
+## Disclaimers
+
+* Both require a "complete" [application manifest](http://docs.cloudfoundry.org/devguide/deploy-apps/manifest.html) (which needs to include all of your service bindings, etc.) &mdash; if you don't have one yet, generate one from an existing app:
+
+    ```bash
+    cf create-app-manifest <APP_NAME> -p manifest.yml
+    ```
+
+* Both will spin up an instance of your environment uploading your **local** code.
+    * Note that this may not be in sync with your live app.
+    * You can leverage this fact to include files that your one-off task might need, such as a data file for importing.
+
+## Short-lived app
+
+The idea here is that we are going to deploy a new application, but running the one-off task, rather than starting a server or whatever it would normally do. Note that this will not work for any command that is interactive.
+
+1. Make a copy of your application manifest:
+
+    ```bash
+    cp manifest.yml task_manifest.yml
+    ```
+
+1. Modify the `task_manifest.yml`:
+    * Change the `name` value to be `task-runner` (or something descriptive).
+    * Remove the following attributes, if present:
+        * `domain`
+        * `domains`
+        * `host`
+        * `hosts`
+        * `random-route`
+    * Add [`no-route: true`](https://docs.cloudfoundry.org/devguide/deploy-apps/manifest.html#no-route).
+    * Set [`command`](https://docs.cloudfoundry.org/devguide/deploy-apps/manifest.html#start-commands) to be whatever you want to execute.
```

Comment:
What happens when the command exits? We've been adding `&& sleep infinity` to the command to prevent it from re-running on complete. Seems like cf will restart the command forever otherwise.